.github/workflows/lint-test.yaml

```diff
@@ -83,6 +83,10 @@ jobs:
           - name: Horizontal Pod Autoscaling Enabled
             helm_args: '--helm-extra-set-args "--values charts/nextcloud/test-values/hpa.yaml"'
 
+          # test the helm chart with notify push enabled
+          - name: Notify Push Enabled
+            helm_args: '--helm-extra-set-args "--values charts/nextcloud/test-values/notify_push.yaml"'
```

Comment:
So we only test that the deployment succeeds, could we use the notify_push self-test check (or whatever it is called) to ensure it works correctly?
Or would that already be covered automatically because the deployments won't succeed otherwise?